src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/migration/MigrateVideoBookingServiceTest.kt

```diff
@@ -668,6 +668,114 @@ class MigrateVideoBookingServiceTest {
+    )
+
+    service.migrate(
+      VideoBookingMigrateResponse(
+        videoBookingId = 5,
+        offenderBookingId = 1,
+        prisonCode = WERRINGTON,
+        courtCode = BLACKPOOL_MC_PPOC,
+        probation = true,
+        createdBy = "MIGRATION PROBATION USER",
+        madeByTheCourt = true,
+        comments = "Migrated probation comments",
+        main = AppointmentLocationTimeSlot(1, yesterday(), LocalTime.of(10, 0), LocalTime.of(11, 0)),
+        cancelled = false,
+        events = listOf(
+          VideoBookingEvent(
+            eventId = 1,
+            eventTime = 2.daysAgo().atStartOfDay(),
+            eventType = VideoLinkBookingEventType.CREATE,
+            comment = "Probation booking create event comments",
+            courtCode = BLACKPOOL_MC_PPOC,
+            preStartTime = null,
+            preEndTime = null,
+            mainStartTime = yesterday().atTime(10, 0),
+            mainEndTime = yesterday().atTime(11, 0),
+            postStartTime = null,
+            postEndTime = null,
+            prisonCode = WERRINGTON,
+            createdByUsername = "MIGRATION PROBATION CREATE USER",
+            courtName = null,
+            madeByTheCourt = true,
+            preLocationId = null,
+            mainLocationId = 1,
+            postLocationId = null,
+          ),
+          VideoBookingEvent(
+            eventId = 2,
+            eventTime = 1.daysAgo().atStartOfDay(),
+            eventType = VideoLinkBookingEventType.UPDATE,
+            comment = "Probation booking updated event comments",
+            courtCode = BLACKPOOL_MC_PPOC,
+            preStartTime = null,
+            preEndTime = null,
+            mainStartTime = yesterday().atTime(10, 0),
+            mainEndTime = yesterday().atTime(11, 0),
+            postStartTime = null,
+            postEndTime = null,
+            prisonCode = WERRINGTON,
+            createdByUsername = "MIGRATION PROBATION UPDATE USER",
+            courtName = null,
+            madeByTheCourt = true,
+            preLocationId = null,
+            mainLocationId = 1,
+            postLocationId = null,
+          ),
+        ),
+      ),
+    )
+
+    inOrder(videoBookingRepository, bookingHistoryRepository) {
+      verify(videoBookingRepository).saveAndFlush(videoBookingCaptor.capture())
+      verify(bookingHistoryRepository, times(2)).saveAndFlush(bookingHistoryCaptor.capture())
+    }
+
+    bookingHistoryCaptor.allValues hasSize 2
+
+    with(bookingHistoryCaptor.firstValue) {
+      historyType isEqualTo HistoryType.CREATE
+      createdBy isEqualTo "MIGRATION PROBATION CREATE USER"
+      createdTime isEqualTo 2.daysAgo().atStartOfDay()
+
+      appointments().single()
+        .isAtPrison(WERRINGTON)
+        .isForPrisonerNumber("DEF123")
+        .isOnDate(yesterday())
+        .startsAt(LocalTime.of(10, 0))
+        .endsAt(LocalTime.of(11, 0))
+        .isForAppointmentType("VLB_PROBATION")
+        .isAtLocation(werringtonLocation)
```

Comment:
Not sure about the appointment type?  Even probation appointments should still be of type `Video Link - Court hearing` ("VLB")  until we put in specific probation journeys later?

Reply:
I take it back. Correct as is.